data/payment_methods.json

```diff
@@ -5168,6 +5168,102 @@
       "en": "Yes Bank"
     }
   },
+  {
+    "code": "allahabad_bank",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "Allahabad Bank"
+    }
+  },
+  {
+    "code": "corporation_bank",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "Corporation Bank"
+    }
+  },
+  {
+    "code": "oriental_bank_of_commerce",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "Oriental Bank Of Commerce"
+    }
+  },
+  {
+    "code": "city_union_bank",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "City Union Bank"
+    }
+  },
+  {
+    "code": "vijaya_bank",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "Vijaya Bank"
+    }
+  },
+  {
+    "code": "united_bank_of_india",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "United Bank Of India"
+    }
+  },
+  {
+    "code": "state_bank_of_bikaner_and_jaipur",
+    "category": "bank_transfer",
+    "status": "active",
+    "name": {
+      "en": "State Bank Of Bikaner And Jaipur"
+    }
+  },
+  {
+    "code": "ratnakar_bank",
```

Comment:
RBL Bank

RBL Bank, formerly known as Ratnakar Bank, is an Indian private sector bank headquartered in Mumbai and founded in 1943.